src/pages/DiaryCard/DiaryCard.jsx

```diff
@@ -1,29 +1,17 @@
+import { useNavigate } from "react-router-dom";
 import styles from "../DiaryCard/DiaryCard.module.css";
-import { MdModeEdit, MdDelete } from "react-icons/md";
 
-export default function DiaryCard({ diary, onDelete, onEdit }) {
+export default function DiaryCard({ diary, onDelete }) {
```

Comment:
You don't need `onDelete` prop any more

Reply:
removed the unnecessary 'onDelete' prop

Reply:
resolved by [eab60fc](https://github.com/Gracekangdh/DiaryApp/pull/1/commits/eab60fc9b08f005d34e90e596d6061603365c022)